frontend/src/views/Nodes/NodeChainList/NodeChainList.js

```diff
@@ -79,10 +80,7 @@ class NodeChainList extends Chains {
           <Col className="page-title">
             <h1>Choose chains</h1>
             <p>
-              Choose the chains you want to connect your node to. Remember you
-              won&#39;t be able to change these chains unless you unstake then
-              restake which will be evenly divided across the selected number of
-              chains.
+              Choose the chains you want to connect your app or node to. Remember you won&#39;t be able to change these chains until your next stake which will be evenly divided on the selected number of chains.
             </p>
           </Col>
```

Comment:
Choose the chains that your Pocket Node will service. Your staked POKT will be evenly divided between these selections. You will not be able to change this selection unless you unstake and restake your node.